modules/xfem/src/efa/EFAElement3D.C

```diff
@@ -1741,11 +1840,20 @@ unsigned int
 EFAElement3D::numFaceNeighbors(unsigned int face_id) const
 {
   unsigned int num_neighbors = 0;
-  if (_face_neighbors[face_id][0])
+  if (_face_neighbors[face_id].size())
```

Comment:
This whole function can be simplified further to just be a one-liner:
`return _face_neighbors[face_id].size();`